lisa/sut_orchestrator/azure/common.py

```diff
@@ -193,9 +206,60 @@ class AzureVmPurchasePlanSchema:
     publisher: str
 
 
+class ArchitectureType(str, Enum):
```

Comment:
I remember there are some similar declared, please reuse them. If not exist, move it to schema.py for reusing.

Reply:
It is not created, it is moved from azure/features.py to prevent import loop.

Reply:
It's a general setting, and will be used by all platforms. So it makes sense to be in schema.py.